src/libs/antares/study/parameters.cpp

```diff
@@ -201,12 +233,24 @@ void Parameters::resetSeeds()
     for (auto i = (uint)seedTsGenLoad; i != seedMax; ++i)
         seed[i] = (s += increment);
 }
+void Parameters::resetThresholdsAdqPatch()
+{
+    // Initialize all thresholds values for adequacy patch
+    adqPatch.curtailmentSharing.thresholdInitiate
+      = adqPatchDefaultValueThresholdInitiateCurtailmentSharingRule;
+    adqPatch.curtailmentSharing.thresholdDisplayViolations
+      = adqPatchDefaultValueThresholdDisplayLocalMatchingRuleViolations;
+}
 
 void Parameters::resetAdqPatchParameters()
 {
     adqPatch.enabled = false;
     adqPatch.localMatching.setToZeroOutsideInsideLinks = true;
     adqPatch.localMatching.setToZeroOutsideOutsideLinks = true;
+    adqPatch.curtailmentSharing.priceTakingOrder = Data::AdequacyPatch::AdqPatchPTO::isDens;
+    adqPatch.saveIntermediateResults = false;
```

Comment:
This seems to be completely unused, remove ? This remark applies to variables in the problem, `Parameters`, and buttons in the GUI.

Reply:
Forget it